packages/react-core/src/components/Pagination/PaginationOptionsMenu.tsx

```diff
@@ -56,6 +56,8 @@ export interface PaginationOptionsMenuProps extends React.HTMLProps<HTMLDivEleme
   containerRef?: React.RefObject<HTMLDivElement>;
   /** @beta The container to append the pagination options menu to. Overrides the containerRef prop. */
   appendTo?: HTMLElement | (() => HTMLElement) | 'inline';
+  /** Flag indicating if scroll on focus of the first menu item should occur. */
+  preventScrollOnItemFocus?: boolean;
```

Comment:
nit. Same comment here about the name of this prop. Can we prefix it with `should`?